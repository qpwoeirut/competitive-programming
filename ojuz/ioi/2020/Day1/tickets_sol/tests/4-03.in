df804a0e-e4b1-4b8d-a0f5-83401a72b5a3
10 10 10
101126342 180396126 534866484 624519351 635550932 648571032 699891055 824842146 941222848 955648325
34115604 44944719 238339215 387361103 436849817 490870946 628475291 635677202 804560479 937027553
218099078 223568865 327964329 431164610 442562958 456538057 487517599 814507450 869338484 890014262
35533914 135972476 292593192 344979960 478890968 563884840 567266829 636458299 655180180 771537347
167023548 281859320 299533430 524872808 601194377 655740965 776170124 837065630 889095396 994590350
364681809 407434545 430468349 436900771 504920397 578968517 658006048 761877309 774219527 936866573
6616463 73666533 176030130 274716816 364270696 416424457 841983273 862991232 902962925 974299608
11599689 44517720 251630920 472113322 616660654 649209066 744702812 829180302 917282309 943896728
186532688 286532893 380754991 396563230 414596737 542449521 630406334 710103381 813047082 900954737
112194322 230512738 236324342 317030402 372499064 381270151 773584678 957967893 974257560 982538877
