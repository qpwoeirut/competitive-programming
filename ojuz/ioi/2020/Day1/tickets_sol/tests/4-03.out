131b432f-1f84-4f8f-9d55-41e868a6bc5d
OK
24057831018
6 8 0 1 2 3 4 5 7 9
0 1 2 4 6 8 3 5 7 9
0 1 2 3 4 6 8 5 7 9
0 2 4 6 8 1 3 5 7 9
2 4 6 8 0 1 3 5 7 9
1 3 5 7 9 0 2 4 6 8
0 1 3 5 7 9 2 4 6 8
3 5 7 9 0 1 2 4 6 8
1 3 5 7 9 0 2 4 6 8
0 2 4 5 7 9 1 3 6 8
